simulators/main/oeeAPI.py

```diff
@@ -174,23 +174,25 @@ def get_shiftplan(self, locationId, dateFrom, dateTo):
         log.warning(f'Cannot get shiftplan for {locationId}, url: {url},  response: {response.status_code}: {response.text} ')
         return {'locationId':locationId,'timeslots':{}}
 
-    def create_or_update_asset_hierachy(self, deviceIDs):
+    def create_or_update_asset_hierarchy(self, deviceIDs):
         line_description = "Simulator LINE"
         lineHierarchy = []
         for deviceID in deviceIDs:
             profileID = self.c8y_api.get_profile_id(deviceID=deviceID)
             if profileID != "":
                 lineHierarchy.append({"profileID":profileID, "ID":deviceID})
 
-        line_id = self.get_id_of_asset_hierachy_line(line_description)
+        line_id = self.get_id_of_asset_hierarchy_line(line_description)
         if line_id == '':
-            line_id = self.c8y_api.createISAType(type="LINE", hierachy=None, description=line_description, oeetarget=80)
-            self.c8y_api.createISAType(type="SITE", hierachy=[{"profileID": None, "ID": line_id}], description="Simulator SITE", oeetarget=80)
+            line_id_dict = self.c8y_api.createISAType(type="LINE", hierarchy=None, description=line_description, oeetarget=80)
```

Comment:
maybe we can find a better name for this? so e.g. "line_id_ISA_Type" instead of "line_id_dict"?